h/accounts/schemas.py

```diff
@@ -203,13 +203,15 @@ class SocialLoginSignupSchema(colander.Schema):
     comms_opt_in = comms_opt_in_node()
 
 
-class EmailChangeSchema(CSRFSchema):
-    email = email_node(title=_("Email address"))
-    # No validators: all validation is done on the email field
-    password = password_node(title=_("Confirm password"), hide_until_form_active=True)
+class EmailAddSchema(colander.Schema):
+    email = email_node()
+
+
+class EmailChangeSchema(colander.Schema):
+    email = email_node()
+    password = password_node()
 
     def validator(self, node, value):
-        super().validator(node, value)
```

Comment:
Now that this class no longer inherits from `CSRFSchema` there's no base `__init__()` to call with these arguments (it would crash).